test/uk/gov/hmrc/gform/dms/DmsSubmissionControllerSpec.scala

```diff
@@ -17,25 +17,26 @@
 package uk.gov.hmrc.gform.dms
 
 import java.time._
-import java.util.{ Base64, UUID }
+import java.util.{Base64, UUID}
 
 import org.apache.pdfbox.pdmodel.PDDocument
-import play.api.libs.json.{ JsValue, Json }
+import play.api.libs.Files.{SingletonTemporaryFileCreator, TemporaryFile}
```

Comment:
This will break a build since it is not formatted by scalafmt. Please format code by scalafmt first.